packages/endpoints-overlay/package.json

```diff
@@ -9,7 +9,7 @@
   "homepage": "https://github.com/opentripplanner/otp-ui#readme",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "tsc": "tsc && echo Remove the @ts-expect-error exception in index.tsx once core-utils is typed!"
```

Comment:
I would wait for the new core-utils version number before updating this package.

Reply:
I agree. This is why I haven't made a fix or feat commit for these packages.